consensus/validator.go

```diff
@@ -156,6 +155,16 @@ func (consensus *Consensus) onPrepared(msg *msg_pb.Message) {
 	consensus.mutex.Lock()
 	defer consensus.mutex.Unlock()
 
+	if consensus.BlockVerifier == nil {
```

Comment:
If we do change like this, there will not be any unverified block in FBFTLog, which means there will not be last mile block in consensus cached to be added in `consensus.AddLastMile`. This could potentially cause the sync never catch up with the latest consensus if missed one block. Is this what we expect?

Reply:
ok, got it. thanks.